esquery.js

```diff
@@ -121,21 +119,21 @@ function matches(node, selector, ancestry) {
                         case 'literal': return '' + selector.value.value === '' + p;
                         case 'type': return selector.value.value === typeof p;
                     }
-                    throw new Error("Unknown selector value type: " + selector.value.type);
+                    throw new Error('Unknown selector value type: ' + selector.value.type);
```

Comment:
Can we get away with using templates? Looks like they're supported in node 8.

Reply:
Sure. I've added a rule to enforce this as well: https://eslint.org/docs/rules/prefer-template